packages/cozy-pouch-link/package.json

```
@@ -18,6 +18,7 @@
   },
   "devDependencies": {
     "babel-cli": "6.26.0",
+    "jest-localstorage-mock": "^2.3.0",
```

si tu peux fixé la version sinon c'est renovate qui fera une PR ;)
```suggestion
    "jest-localstorage-mock": "2.3.0",
```